docs/administration-guide/security/user-control.md

```diff
@@ -1,3 +1,50 @@
+# User Authentication
+
 !!! info "Work in Progress"
 
-    This page is under construction.
\ No newline at end of file
+    This page is under construction.
+
+The user authentication layer for Gaffer is currently only enforced by the REST
+API. We recommend restricting users such that they do not have access to the
+underlying Java API so that all queries are authenticated and executed via the
+REST API.
+
+In the REST API the `User` object is constructed via a [`UserFactory`](https://gchq.github.io/Gaffer/uk/gov/gchq/gaffer/rest/factory/UserFactory.html).
+In the Spring REST API an abstract implementation of this class is used,
+`AbstractUserFactory`, which is then used in the passing of HTTP headers for
+authentication.
+
+Currently, there is a single default implementation of this; the
+`UnknownUserFactory` which simply returns a new `User` with `UNKNOWN` as the
+user ID. The `UnknownUserFactory` is the default factory, to specify the user
```

Comment:
```suggestion
user ID. To specify the user
```
Not sure you need to repeat that UnknownUserFactory is the default as you have already said that in the previous sentence